.github/workflows/master-to-main.yml

```diff
@@ -0,0 +1,77 @@
+name: Convert the master branch to the main branch and set up a monorepo
+
+on:
+  workflow_dispatch:
+
+jobs:
+  master-to-main:
+    if: github.triggering_actor == "nickvigilante"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.5.0
+        with:
+          ref: master-2
+          path: source
+      - uses: actions/checkout@v3.5.0
+        with:
+          ref: main-2
+          path: target
+      - name: Copy new file changes from master to main
+        run: |
+          cp -f source/v1.0 target/docs/docs
+          cp -f source/v1.1 target/docs/docs
+          cp -f source/v2.0 target/docs/docs
+          cp -f source/v2.1 target/docs/docs
+          cp -f source/v19.1 target/docs/docs
+          cp -f source/v19.2 target/docs/docs
+          cp -f source/v20.1 target/docs/docs
+          cp -f source/v20.2 target/docs/docs
+          cp -f source/v21.1 target/docs/docs
+          cp -f source/v21.2 target/docs/docs
+          cp -f source/v22.1 target/docs/docs
+          cp -f source/v22.2 target/docs/docs
+          cp -f source/v23.1 target/docs/docs
+
+          cp -f source/_data target/docs/docs
+          cp -f source/_includes target/docs/docs
+          cp -f source/_layouts target/docs/docs
+          cp -f source/_plugins target/docs/docs
+
+          cp -f source/advisories target/docs/docs
+          cp -f source/api target/docs/docs
+          cp -f source/cockroachcloud target/docs/docs
+          cp -f source/css target/docs/docs
+          cp -f source/fonts target/docs/docs
+          cp -f source/images target/docs/docs
+          cp -f source/jekyll-algolia-dev target/docs/docs
+          cp -f source/Gemfile target/docs/docs
+
+          cp -f source/js target/docs/docs
+          cp -f source/releases target/docs/docs
+          cp -f source/tutorials target/docs/docs
+
+          cp -f source/Makefile target/docs/docs
+          cp -f source/_config_base.yml target/docs/docs
+          cp -f source/_config_cockroachdb_local.yml target/docs/docs
+          cp -f source/_config_cockroachdb_no_remote_cache.yml target/docs/docs
+          cp -f source/_config_cockroachdb.yml target/docs/docs
+          cp -f source/_redirects target/docs/docs
+
+          cp -f source/404.md target/docs/docs
+          cp -f source/index.md target/docs/docs
+          cp -f source/search.html target/docs/docs
+          cp -f source/sitemap.xml target/docs/docs
```

Comment:
I think this looks right, in terms of what belongs top-level and what belongs within the new `docs` subdir of `docs`, but I'm definitely not the authority here!